### PR TITLE
adds prometheus-pingdom-exporter chart

### PR DESCRIPTION
**Pull request by monotek**
#### What this PR does / why we need it:
* adds prometheus-pingdom-exporter chart from https://github.com/kiwigrid/helm-charts which will be deprecated in https://github.com/kiwigrid/helm-charts/pull/416

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by monotek** [comment]
Sorry for the review request to everybody :(
Was done by following the dco signing steps.

File: charts/prometheus-pingdom-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+keywords:
+- prometheus
+- pingdom
+sources:
+- https://github.com/giantswarm/prometheus-pingdom-exporter
+- https://github.com/camptocamp/prometheus-pingdom-exporter
+maintainers:
+- name: monotek
+  email: monotek23@gmail.com
+icon: https://raw.githubusercontent.com/cncf/artwork/master/prometheus/icon/color/prometheus-icon-color.svg
```
Comment (gkarthiks) [inline charts/prometheus-pingdom-exporter/Chart.yaml:16]: @monotek can we also add the artifacthub annotations.

/cc @scottrigby

**Review by torstenwalter — APPROVED** [review]

**Comment by monotek** [comment]
To be honest I'm not sure how to proceed here? It looks like the dco failures are in the commits which are not mine. Should i go for the dco override option?

**Comment by monotek** [comment]
@torstenwalter 
i used the override dco option now but i could only squash the commits.
Can you preserve the commits of the old contributers while merging?

**Comment by torstenwalter** [comment]
@monotek iI temporarily allowed merge commits and merged it using that.

**Comment by monotek** [comment]
Thanks :)
Does @rpahli gets automatically an invite to the helm charts github group to be able to co maintain?

**Pull request merged**